arch/arm64/configs/j6lte_defconfig

```diff
@@ -1575,7 +1575,7 @@ CONFIG_WLAN_QCA9377_DUAL_BDF=0
 # CONFIG_WCNSS_MEM_PRE_ALLOC is not set
 # CONFIG_ATH_CARDS is not set
 # CONFIG_BCM4330 is not set
-# CONFIG_BCM4343 is not set
```

Comment:
Didnt notice i Accidentally Disabled the wifi driver 😆

Reply:
lol